voluncheer/voluncheer/settings.py

```diff
@@ -91,6 +94,39 @@
 ]
 
 WSGI_APPLICATION = "voluncheer.wsgi.application"
+ASGI_APPLICATION = "voluncheer.asgi.application"
+
+redis_prod_url = "redis://chatroom-redis-cluster-prod.byxkua.ng.0001.use1.cache.amazonaws.com:6379"
```

Comment:
For environment specific variables, set them in the environment using variables and read them in here. See the various other cases in here for examples.